pkg/loop/plugin_relayer.go

```diff
@@ -6,15 +6,17 @@ import (
 	"github.com/hashicorp/go-plugin"
 	"google.golang.org/grpc"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/net"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer"
+	internal "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer"
+	looptypes "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/types"
```

Comment:
Double import of `package relayer` as `internal`, which could be used for `internal/types`:
```suggestion
	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer"
	internal "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/types"
```